test-utils/src/main/java/org/javaee7/ServerOperations.java

```diff
@@ -63,63 +63,63 @@ public static void addUsersToContainerIdentityStore() {
                 System.out.println(javaEEServer + " not supported");
             }
         }
-        
+
         // TODO: support other servers than Payara and GlassFish
-        
+
         // WildFly ./bin/add-user.sh -a -u u1 -p p1 -g g1
     }
-    
+
     public static void addCertificateToContainerTrustStore(Certificate clientCertificate) {
-        
+
         String javaEEServer = System.getProperty("javaEEServer");
-        
+
         if ("glassfish-remote".equals(javaEEServer) || "payara-remote".equals(javaEEServer)) {
-            
-            String gfHome = System.getProperty("glassfishRemote_gfHome");
+
+            String gfHome = System.getProperty("com.sun.aas.productRoot");
```

Comment:
Does this work? 🤔
That's a Payara/GlassFish system property - not typically one available or specified outside of a Payara runtime.
`glassfishRemote_gfHome` is explicitly used in EE7 samples

Reply:
You are way behind on this :) This has been removed / added  long ago